ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Nti/EditableNTIPageModel.cs

```diff
@@ -0,0 +1,66 @@
+using ConcernsCaseWork.API.Contracts.Constants;
+using ConcernsCaseWork.Extensions;
+using ConcernsCaseWork.Models;
+using ConcernsCaseWork.Models.Validatable;
+using ConcernsCaseWork.Pages.Base;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ConcernsCaseWork.Pages.Case.Management.Action.Nti;
+
+[BindProperties]
+public class EditableNTIPageModel : AbstractPageModel
```

Comment:
SUGGESTION: I would call this CloseNtiBasePage

Its a page that is used by all close states of an NTI